reactDashboard/src/components/Footer/Footer.css

```diff
@@ -66,3 +66,11 @@
 footer {
   padding-bottom: 30px;
 }
+
+.darkmode footer {
```

Comment:
The footer doesn't declare a (normal mode) background-color - do you need this clause?

Reply:
Without this, there is white bleeding through below the footer component; same for the ".darkmode header"